tpot/operator_utils.py

```diff
@@ -243,7 +243,7 @@ def export(cls, *args):
                 # To make sure the inital operators is the first parameter just
                 # for better persentation
                 for dep_op_pname, dep_op_str in dep_op_list.items():
-                    if dep_op_str == 'f_classif':
+                    if dep_op_str in ['f_classif', 'f_regression']:
```

Comment:
Is there a more general way that we can handle this situation? Programming in specific function names doesn't seem to be ideal. We'll run into the same issue with SelectPercentile for other scoring functions as well.